src/reducers/Reducers.js

```diff
@@ -3,17 +3,37 @@ import { combineReducers } from 'redux'
 import { reducer as formReducer } from 'redux-form'
 import LoginConstants from '../constants/LoginConstants.js'
 
-const initialState = {
+const appInitialState = {
 }
 
-const appState = (state: Object = initialState, action: Object) => {
+const loginInitialState = {
+  authToken: null,
```

Comment:
We already have the backend set up. It uses the same thing (JWT) as it did last semester